components/brave_extension/extension/brave_extension/components/advancedView/header.tsx

```diff
@@ -71,8 +70,8 @@ export default class Header extends React.PureComponent<Props, {}> {
     return getTotalBlockedSizeStrings(this.blockedItemsSize, httpsUpgrades)
   }
 
-  onToggleShields = (event: React.ChangeEvent<HTMLInputElement>) => {
-    const shieldsOption: BlockOptions = getToggleStateViaEventTarget(event)
+  onToggleShieldsMain = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const shieldsOption: BlockOptions = event.target.checked ? 'allow' : 'block'
```

Comment:
this is exactly what `getToggleStateViaEventTarget` does. can we keep it?

Reply:
`getToggleStateViaEventTarget` was super confusing to me because in all other cases it returns the wrong value and then has to be switched back with `toggleShieldsValue`